includes/class-wc-google-analytics.php

```diff
@@ -422,6 +423,22 @@ public function load_admin_assets() {
 		);
 	}
 
+	/**
+	 * Add suggested privacy policy content
+	 *
+	 * @return void
+	 */
+	public function privacy_policy() {
+		$content = '<p class="privacy-policy-tutorial">' . sprintf(
```

Comment:
Same in https://github.com/woocommerce/google-listings-and-ads/pull/2106, this class will cause the `Copy suggested policy text to clipboard` button to not function correctly.